src/ChannelServer/World/Skill.cs

```diff
@@ -87,10 +115,9 @@ public Skill(Character character, int skillId, int level)
 			this.Character = character;
 			this.Id = skillId;
 			this.Level = level;
-			this.OverHeatCount = 0; // So they don't reuse this value just cooldown value?
 			this.Data = ChannelServer.Instance.Data.SkillDb.Find(skillId) ?? throw new ArgumentException($"Unknown skill '{skillId}'.");
-			this.CoolDownData = ChannelServer.Instance.Data.CoolDownDb.Find(this.Data.CoolDownGroup) ?? throw new ArgumentException($"Unknown skill '{skillId}' cooldown group '{this.Data.CoolDownGroup}'.");
-			this.OverHeatData = ChannelServer.Instance.Data.CoolDownDb.Find(this.Data.OverHeatGroup) ?? throw new ArgumentException($"Unknown skill '{skillId}' overheat group '{this.Data.OverHeatGroup}'.");
+			this.CooldownData = ChannelServer.Instance.Data.CoolDownDb.Find(this.Data.CooldownGroup) ?? throw new ArgumentException($"Unknown skill '{skillId}' cooldown group '{this.Data.CooldownGroup}'.");
```

Comment:
Didn't we use Cool**D**own everywhere? Honestly, I never know whether I should write Cooldown or CoolDown. "The cooldown", as a thing, feels kind of correct. What do you say? 🤔

Reply:
I hit the good ole google to double-check. I went with Cooldown since we're fixing the OverHeat to Overheat.
https://www.merriam-webster.com/dictionary/cooldown

Reply:
Yea, I assumed that was the reason. If you search both "cooldown" and "cool down", the latter has more results, but then there's dictionaries... Let's just go with your change^^

Reply:
Oh! Wikipedia actually differentiates between the two, saying cooldown is a gaming term and cool down an excercise.